src/app.ts

```diff
@@ -21,17 +19,22 @@ import https from 'https';
 import WebSocket from 'ws';
 import config from 'config';
 
-if (process.env.NODE_ENV === 'development') {
-    const httpsAgent = new https.Agent({
-        rejectUnauthorized: false,
-    });
+// Manager
+import processEnvManager from './manager/processEnv/processEnv.manager';
+import { logger } from './utils/logger';
+
+// To discuss
```

Comment:
Choose a common tag for those comments: for example i'm pretty sure there are some vscode extensions that recognize tags like //TODO: or //@TODO: just chose one so we can start to use one of those tags